Alpine:3.7 + GCC 6.4.0 build failed

OS / GCC:

```
# gcc -###
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-alpine-linux-musl/6.4.0/lto-wrapper
Target: x86_64-alpine-linux-musl
Configured with: /home/buildozer/aports/main/gcc/src/gcc-6.4.0/configure --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --build=x86_64-alpine-linux-musl --host=x86_64-alpine-linux-musl --target=x86_64-alpine-linux-musl --with-pkgversion='Alpine 6.4.0' --enable-checking=release --disable-fixed-point --disable-libstdcxx-pch --disable-multilib --disable-nls --disable-werror --disable-symvers --enable-__cxa_atexit --enable-default-pie --enable-cloog-backend --enable-languages=c,c++,objc,java,fortran,ada --disable-libssp --disable-libmpx --disable-libmudflap --disable-libsanitizer --enable-shared --enable-threads --enable-tls --with-system-zlib --with-linker-hash-style=gnu
Thread model: posix
gcc version 6.4.0 (Alpine 6.4.0)


/opt/tarantool-c # uname -a
Linux b7c37b90c2fb 4.9.87-linuxkit-aufs #1 SMP Wed Mar 14 15:12:16 UTC 2018 x86_64 Linux
/opt/tarantool-c # cat /etc/*-release
3.7.0
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.7.0
```


Build configuration: 
```
-- ------------------------------------------------
--         Tarantool library configuration:
-- ------------------------------------------------
--   CMAKE_BUILD_TYPE: Debug
--   PREFIX: /usr/local
--   C_COMPILER: /usr/bin/cc
--   C_FLAGS: -Wall -Wextra -std=gnu99 -D__USE_XOPEN
```
Error:
```
In file included from /opt/tarantool-c/tnt/tnt_buf.c:37:0:
/opt/tarantool-c/include/tarantool/tnt_reply.h:53:18: error: expected declaration specifiers or '...' before '*' token
 typedef ssize_t (*tnt_reply_t)(void *ptr, char *dst, ssize_t size);
                  ^
/opt/tarantool-c/include/tarantool/tnt_reply.h:138:37: error: unknown type name 'tnt_reply_t'
 tnt_reply_from(struct tnt_reply *r, tnt_reply_t rcv, void *ptr);
                                     ^~~~~~~~~~~
```


Suggested patch:
```
diff --git tnt/tnt_buf.c tnt/tnt_buf.c
index 0bcbb00..2fb9abc 100644
--- tnt/tnt_buf.c
+++ tnt/tnt_buf.c
@@ -31,7 +31,7 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
-
+#include <sys/types.h>
 #include <tarantool/tnt_mem.h>
 #include <tarantool/tnt_proto.h>
 #include <tarantool/tnt_reply.h>
```

--- a/tnt/tnt_buf.c
+++ b/tnt/tnt_buf.c
@@ -31,7 +31,7 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
-
+#include <sys/types.h>
 #include <tarantool/tnt_mem.h>
 #include <tarantool/tnt_proto.h>
 #include <tarantool/tnt_reply.h>

